app/templates/gentelella/users/events/index.html

```diff
@@ -91,7 +91,18 @@
     <script src="{{ url_for('static', filename='vendor/datatables.net-bs/js/dataTables.bootstrap.min.js') }}"></script>
     <script src="{{ url_for('static', filename='vendor/bootstrap-progressbar/bootstrap-progressbar.min.js') }}"></script>
     <script type="text/javascript">
-        $('.table').DataTable();
+
+
+
+  $('.table').DataTable( {"columnDefs": [
```

Comment:
Please fix the alignment and indentation ... Always ensure a consistent code-style is followed. Let the keys start from a new line ... Align the indentation ...